src/analysis/retail/priest/discipline/modules/Abilities.ts

```diff
@@ -126,6 +126,18 @@ class Abilities extends CoreAbilities {
           suggestion: true,
         },
```

Comment:
Dont forget to also update Shadowfiend and Mindbender to ensure they're disabled when Voidwraith is enabled.